providers/os/connection/snapshot/blockdevices.go

```diff
@@ -125,29 +126,59 @@ func (blockEntries BlockDevices) GetMountablePartitionByDevice(device string) (*
 	return &PartitionInfo{Name: devFsName, FsType: partitions[0].FsType}, nil
 }
 
-// Searches for a device by name
-func (blockEntries BlockDevices) FindDevice(name string) (BlockDevice, error) {
-	log.Debug().Str("device", name).Msg("searching for device")
-	var secondName string
-	if strings.HasPrefix(name, "/dev/sd") {
-		// sdh and xvdh are interchangeable
-		end := strings.TrimPrefix(name, "/dev/sd")
-		secondName = "/dev/xvd" + end
+func lms(lmsCache map[string]int, s1, s2 string) int {
```

Comment:
can you add some doc to this function: whats the cache and the return val? also can you extend the name? its not clear to me what `lms` stands for just by looking at it